docs/design/expressions/pointer_operators.md

```diff
@@ -0,0 +1,54 @@
+# Pointer operators
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Details](#details)
+    -   [Precedence](#precedence)
+-   [Alternatives considered](#alternatives-considered)
+-   [References](#references)
+
+<!-- tocstop -->
+
+## Overview
+
+Carbon provides three unary operators related to pointers:
+
+-   `&` as a prefix unary operator takes the address of an object, forming a
+    pointer to it.
+-   `*` as a prefix unary operator dereferences a pointer.
```

Comment:
Did you consider making the above postfix operations? Circle experimented with this and this ended up making syntax for common code require fewer parenthesis and easier to read.